include/polycap-photon.h

```diff
@@ -118,6 +118,13 @@ polycap_vector3 polycap_photon_get_exit_electric_vector(polycap_photon *photon);
  */
 void polycap_photon_free(polycap_photon *photon);
 
+/** Free a polycap_photon_leaks structure
+ * 
+ * \param leak a polycap_photon_leaks structure
+ * \param n_leaks the amount of leak events contained in the polycap_photon_leaks structure
+ */
+void polycap_leaks_free(polycap_leaks *leak, int64_t n_leaks);
```

Comment:
Maak deze functie intern door ze in `polycap-private.h` te zetten: de gebruiker kan zover ik zie toch toch niet aan een `polycap_leaks` struct geraken met behulp van de beschikbare API.

Nog iets: ik denk dat ge beter `polycap_leaks` hernoemt naar `polycap_leak` aangezien een zo een struct enkel informatie bevat over een enkel leak? Als ik mij niet vergis tenminste...